converter/tree.py

```diff
@@ -83,8 +83,10 @@ def get_node_type(fig_node: dict, parent_type: str) -> str:
     # We do this because Sketch does not support nested artboards
     # If a Frame is detected inside another Frame, the internal one
     # is considered a group
-    if fig_node["type"] == "FRAME":
-        if parent_type == "CANVAS" and not fig_node["resizeToFit"]:
+    if fig_node["type"] == "FRAME" or fig_node["type"] == "SECTION":
+        if parent_type == "CANVAS" and (
+            "resizeToFit" not in fig_node or not fig_node["resizeToFit"]
```

Comment:
```suggestion
            not fig_node.get("resizeToFit")
```

This also allow removing the wrapping parenthesis and maybe it fits in a single line

Reply:
https://github.com/sketch-hq/fig2sketch/pull/63#discussion_r1037933757